heron/ui/src/python/main.py

```diff
@@ -134,14 +136,15 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0' if command_line_args['address'] is None else command_line_args['address']
```

Comment:
@kramasamy you are right 0.0.0.0 is not a good log. Make it clearer,
$ heron-ui, currently it prints Listening at http://{ip address from one of the interfaces}:8889, but it actually LISTEN 0.0.0.0/:::8889